server/vulnerabilities/oval/oval_platform.go

```diff
@@ -19,7 +19,7 @@ var SupportedHostPlatforms = []string{"ubuntu", "rhel", "amzn"}
 // getMajorMinorVer returns the major and minor version of an 'os_version'.
 // ex: 'Ubuntu 20.4.0' => '(20, 04)'
 func getMajorMinorVer(osVersion string) (string, string) {
-	re := regexp.MustCompile(` (?P<major>\d+)\.?(?P<minor>\d+)?\.?(\*|\d+)?$`)
+	re := regexp.MustCompile(` (?P<major>\d+)\.?(?P<minor>\d+)?\.?(\*|\d+)?`)
```

Comment:
this is just curiosity: have you considered using `[/etc/os-release`](https://www.freedesktop.org/software/systemd/man/os-release.html)?

Reply:
@roperzh, there's an issue with how Ubuntu views `os-release` that complicates our processing. Here's some additional [context](https://github.com/fleetdm/fleet/issues/4816#issuecomment-1083554889).

Reply:
oh right, I remember now! Thanks.